extensions/machine-learning-services/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+	"compileOnSave": true,
```

Comment:
You should extend the common tsconfig in the folder above (take a look at the git tsconfig).